README.md

```diff
@@ -47,6 +48,7 @@ If you are behind a proxy, configure `https_proxy` environment variable.
 ## Built-in starters
 
 - [app (community-maintained)](https://github.com/stencil-community/stencil-app-starter)
+- [ionic (community-maintained)](https://github.com/stencil-community/stencil-ionic-starter)
```

Comment:
to give this a little more information (and not confuse folks about whether or not this is a community vs ionic led project), can we rename this here to 'ionic-pwa'?

```suggestion
- [ionic-pwa (community-maintained)](https://github.com/stencil-community/stencil-ionic-starter)
```